feat(backend): implement_launchpad_wallet_grpc_client

alex1307 opened the pull request:
Description

Implement wallet grpc cleint and forward the transaction stream to the Tauri front end.
The following events should be available:
	- transaction received
	- transaction sent
	- transaction cancelled
	- transaction mined but unconfirmed

Motivation and Context
The users should be able to see all mined blocks and transactions.

How Has This Been Tested?

Manually. Async subscribe function is created and started along with Launchpad app.
Once the app is started the function tries to connect to the wallet instance and prints all received events/transactions.

alex1307 committed:
feat(backend): impl launchpad wallet grpc client
Description

Implement wallet grpc cleint and forward the transaction stream to the Tauri front end.
The following events should be available:
	- transaction received
	- transaction sent
	- transaction cancelled
	- transaction mined but unconfirmed

Motivation and Context
The users should be able to see all mined blocks and transactions.

How Has This Been Tested?

Manually. Async subscribe function is created and started along with Launchpad app.
Once the app is started the function tries to connect to the wallet instance and prints all received events/transactiosn.

CjS77 commented on applications/launchpad/backend/src/api/mod.rs:
```diff
@@ -55,3 +59,31 @@ pub async fn health_check(image: &str) -> String {
+    let app_clone = app.clone();
+    tauri::async_runtime::spawn(async move {
+        while let Some(response) = stream.next().await {
+            let value = response.transaction.unwrap();
```
Can this fail?
- If not, please put a comment in to indicate this.
- If so, handle the error and return gracefully.

alex1307 replied:
This is a bug and I will fix it.

CjS77 reviewed (commented):
Looks great.

Picked up one potential source of panicking a thread.

alex1307 committed:
feat(backend): impl launchpad wallet grpc client
Description

Fixing code quality issues:
Fixing potential source of panicking a thread.

Motivation and Context
Clean code quality requirement.

How Has This Been Tested?
The change is tested manually.

CjS77 reviewed (approved):
+1
